mmv1/third_party/terraform/services/resourcemanager/resource_google_project_iam_member_remove_test.go

```diff
@@ -0,0 +1,269 @@
+			},
+			{
+				Config:   testAccCheckGoogleProjectIamMemberRemove_basic2(randomSuffix, org),
+				PlanOnly: true, // binding expects the membership to be removed. Any diff will fail the test due to PlanOnly.
+			},
+		},
+	})
+}
+
+func TestAccProjectIamMemberRemove_multipleMembersInBinding(t *testing.T) {
+	t.Parallel()
+
+	org := envvar.GetTestOrgFromEnv(t)
+	randomSuffix := acctest.RandString(t, 10)
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckGoogleProjectIamCustomRoleDestroyProducer(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time": {},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config:             testAccCheckGoogleProjectIamMemberRemove_multipleMemberBinding(randomSuffix, org),
+				ExpectNonEmptyPlan: true, // Due to adding in binding, then removing in remove resource
+			},
+			{
+				Config:   testAccCheckGoogleProjectIamMemberRemove_multipleMemberBinding2(randomSuffix, org),
+				PlanOnly: true, // binding expects the membership to be removed. Any diff will fail the test due to PlanOnly.
+			},
+		},
+	})
+}
+
+func TestAccProjectIamMemberRemove_memberInMultipleBindings(t *testing.T) {
+	t.Parallel()
+
+	org := envvar.GetTestOrgFromEnv(t)
+	randomSuffix := acctest.RandString(t, 10)
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckGoogleProjectIamCustomRoleDestroyProducer(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time": {},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config:             testAccCheckGoogleProjectIamMemberRemove_multipleMemberBinding(randomSuffix, org),
+				ExpectNonEmptyPlan: true, // Due to adding in binding, then removing in remove resource
+			},
+			{
+				Config:   testAccCheckGoogleProjectIamMemberRemove_multipleMemberBinding2(randomSuffix, org),
+				PlanOnly: true, // binding expects the membership to be removed. Any diff will fail the test due to PlanOnly.
+			},
+		},
+	})
+}
+
+func testAccCheckGoogleProjectIamMemberRemove_basic(randomSuffix, org string) string {
+	return fmt.Sprintf(`
+resource "google_project" "project" {
+  project_id = "tf-test-%s"
+  name       = "tf-test-%s"
+  org_id     = "%s"
+}
+
+resource "google_project_iam_binding" "bar" {
+  project = google_project.project.project_id
+  members = ["user:gterraformtest1@gmail.com"]
+  role    = "roles/editor"
+}
+
+resource "time_sleep" "wait_20s" {
+  depends_on = [google_project_iam_binding.bar]
+  create_duration = "20s"
+}
+
+resource "google_project_iam_member_remove" "foo" {
+  role     = "roles/editor"
+  project  = google_project.project.project_id
+  member  = "user:gterraformtest1@gmail.com"
```

Comment:
Do we own the emails used in this test? If not, can we grant a lesser permission than "editor" on the project even though it's short-lived?

Reply:
They're OTAs held by our team, yep. I confirmed that earlier w/ Hao.